stage1/iottymux/iottymux.go

```diff
@@ -410,9 +410,9 @@ func actionIOMux(statusFile string) error {
 	switch logMode {
 	case "k8s-plain":
 		var err error
-		// TODO(lucab): check what should be the target path with Euan
-		logTarget := filepath.Join(pathPrefix, appName, "logfile")
-		logFile, err = os.OpenFile(logTarget, os.O_CREATE|os.O_WRONLY|os.O_TRUNC, os.ModePerm)
+		logPath := os.Getenv("KUBERNETES_LOG_PATH")
+		logFullPath := filepath.Join("/rkt/kubernetes", logPath)
```

Comment:
As this is coming from the outside, I suggest to `Clean` it before use and make sure the result is still within the prefix dir after.